apps/pythonapi/examples/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+set(ALL_NOTEBOOKS "")
+
+function(NOTEBOOK FILE OUT)
+    include(CMakeParseArguments)
+    cmake_parse_arguments(PARSE_ARGV 2
+        "NOTEBOOK"
+        ""
+        ""
+        ""
+        )
+
+    get_filename_component(FILE "${FILE}" REALPATH)
+
+    add_custom_command(
+        OUTPUT "${OUT}"
+        DEPENDS "${FILE}"
+        COMMAND cat "${FILE}" | jupytext --to ipynb > "${OUT}"
+        )
+
+    list(APPEND ALL_NOTEBOOKS "${OUT}")
+    set(ALL_NOTEBOOKS "${ALL_NOTEBOOKS}" PARENT_SCOPE)
+endfunction()
+
+
+# copy(vapor ${PYTHON_API_DIR} EXCLUDE cmake.py __pycache__)
+# copy(setup.py ${PYTHON_API_DIR})
+
+set(EXAMPLE_NOTEBOOK_OUT_DIR "${PYTHON_API_DIR}/example_notebooks")
+file(MAKE_DIRECTORY ${EXAMPLE_NOTEBOOK_OUT_DIR})
+
+copy(example_utils.py ${EXAMPLE_NOTEBOOK_OUT_DIR})
+notebook(numpy_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/numpy_example.ipynb)
+notebook(xarray_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/xarray_example.ipynb)
+notebook(dataset_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/dataset_example.ipynb)
+notebook(flow_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/flow_example.ipynb)
+notebook(annotation_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/annotation_example.ipynb)
+notebook(camera_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/camera_example.ipynb)
+notebook(transfer_function_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/transfer_function_example.ipynb)
+notebook(katrina_example.py ${EXAMPLE_NOTEBOOK_OUT_DIR}/katrina_example.ipynb)
+
+add_custom_target(
+    example_notebooks
+    ALL
+    DEPENDS ${ALL_NOTEBOOKS}
+)
+
+if (CONDA_BUILD)
```

Comment:
My understanding is that CONDA_BUILD prepares Vapor's code base for uploading to the Conda package manager.  When should this be turned on and off?  I'm guessing that if I affect the params or render libraries, this should be on.

Reply:
`CONDA_BUILD` should be turned on when building using conda-build and turned off otherwise.

Reply:
> My understanding is that CONDA_BUILD prepares Vapor's code base for uploading to the Conda package manager.

True or false?

> > When should this be turned on and off? I'm guessing that if I affect the params or render libraries, this should be on.

> CONDA_BUILD should be turned on when building using conda-build and turned off otherwise.

But what is CONDA_BUILD actually building?  This isn't documented and it's not obvious to me.  I know that it can be turned on and off.

Reply:
No? I'm confused, sorry.

Reply:
I guess I just don't understand what CONDA_BUILD/conda-build is actually building/doing.